submissions/HannaSyn/memory-pair-game/app.js

```diff
@@ -0,0 +1,126 @@
+const characters = [
+  {
+    id: 1,
+    backImg: 'assets/Rick.png'
+  },
+  {
+    id: 2,
+    backImg: 'assets/Summer.png'
+  },
+  {
+    id: 3,
+    backImg: 'assets/Morty.png'
+  },
+  {
+    id: 4,
+    backImg: 'assets/missix.png'
+  },
+  {
+    id: 5,
+    backImg: 'assets/Jerry.png'
+  },
+  {
+    id: 6,
+    backImg: 'assets/Beth.png'
+  },
+  {
+    id: 7,
+    backImg: 'assets/pickle-rick.png'
+  },
+  {
+    id: 8,
+    backImg: 'assets/tele.png'
+  }
+]
+
+const playingField = document.querySelector('.playing-field');
+const cards = [...characters, ...characters];
+const cardsCollection = document.getElementsByClassName('card');
+const timeoutCheck = 1000;
+const timeoutReset = 2000;
+
+let firstClicked = null;
+let countCards = cards.length;
+
+const playingFieldBlocked = () => playingField.classList.add('blocked');
```

Comment:
It's better to start the name of the function from a verb.
```suggestion
const blockPlayingField = () => playingField.classList.add('blocked');
```
Or even you can get rid of these functions and just use `playingField.classList.add('blocked')` and `playingField.classList.remove('blocked')`